Build error when I try to use Carthage to involve SwiftHTTP library

My Carthage file:

github "daltoniam/SwiftHTTP" "master"


When I run script: 

carthage update


Error log:

*** Building scheme "SwiftHTTP" in SwiftHTTP.xcodeproj
*** Building scheme "SwiftHTTPOSX" in SwiftHTTP.xcodeproj
2015-06-08 15:56:52.513 xcodebuild[80469:7702550] [MT] PluginLoading: Required plug-in compatibility UUID E969541F-E6F9-4D25-8158-72DC3545A6C6 for plug-in at path '~/Library/Application Support/Developer/Shared/Xcode/Plug-ins/Alcatraz.xcplugin' not present in DVTPlugInCompatibilityUUIDs
2015-06-08 15:56:52.514 xcodebuild[80469:7702550] [MT] PluginLoading: Required plug-in compatibility UUID E969541F-E6F9-4D25-8158-72DC3545A6C6 for plug-in at path '~/Library/Application Support/Developer/Shared/Xcode/Plug-ins/AdjustFontSize.xcplugin' not present in DVTPlugInCompatibilityUUIDs
** BUILD FAILED **


The following build commands failed:
	CompileSwift normal x86_64 /Users/application/myApplication/Carthage/Checkouts/SwiftHTTP/Tests/SwiftHTTPTests.swift
	CompileSwiftSources normal x86_64 com.apple.xcode.tools.swift.compiler
(2 failures)


Does anybody have the same issue?

--- a/Tests/SwiftHTTPTests.swift
+++ b/Tests/SwiftHTTPTests.swift
@@ -7,7 +7,6 @@
 //
 
 import XCTest
-import UIKit
 import SwiftHTTP
 
 class SwiftHTTPTests: XCTestCase {

